features/configuration/read_options_from_file.feature

```diff
@@ -9,12 +9,18 @@ Feature: read command line configuration options from files
     * Project:  `./.rspec` (i.e. in the project's root directory, usually
       checked into the project)
 
-    * Global: `~/.rspec` (i.e. in the user's home directory)
+    * Global (HOME): `~/.rspec` (i.e. in the user's home directory)
 
-  Configuration options are loaded from `~/.rspec`, `.rspec`, `.rspec-local`,
-  command line switches, and the `SPEC_OPTS` environment variable (listed in
-  lowest to highest precedence; for example, an option in `~/.rspec` can be
-  overridden by an option in `.rspec-local`).
+    * Global (XDG): `$XDG_CONFIG_HOME/rspec/options` (i.e. in the user's
+      [the XDG Base Directory
```

Comment:
"in the user's the XDG Base Directory Specification config directory" reads a bit funny.  Maybe remove the 2nd "the"?